examples/cross-platform-builds/skaffold.yaml

```diff
@@ -1,15 +1,13 @@
 apiVersion: skaffold/v2beta29
```

Comment:
nit: likely makes sense to link the cross-platform docs in the README.md in case users investigate `examples/cross-platform-builds` to see how this feature works

Reply:
that makes sense, what is the link to the v2 docs?